cartography/intel/crxcavator/__init__.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import os
+
+from cartography.intel.crxcavator.crxcavator import sync_extensions
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+# API key for CRXcavator generaated from crxcavator.io portal
+CRXCAVATOR_API_KEY = os.environ.get('CREDENTIALS_CRXCAVATOR_API_KEY')
+
+# API for the CRXcavator API - https://api.crxcavator.io/v1 as of 07/09/19
+CRXCAVATOR_API_BASE_URL = os.environ.get('CRXCAVATOR_URL')
+
+
+def start_extension_ingestion(session, config):
+    """
+    If this module is configured, perform ingestion of CRXcavator data. Otherwise warn and exit
+    :param session: Neo4J session for database interface
+    :param config: Neo4J server URI and Update tag for data freshness
+    :return: None
+    """
+    if not CRXCAVATOR_API_BASE_URL or not CRXCAVATOR_API_KEY:
+        logger.debug('CRXcavator import is not configured - skipping this module. See docs to configure.')
```

Comment:
should we print something to console and/or warning?

Reply:
Was originally info, downgraded to debug to match similar pattern in GCP, but noticed it is logged as both debug and error. Changed to warning.